roles/robot-pkgs/tasks/main.yml

```diff
@@ -1,5 +1,9 @@
 ---
 # tasks file for robot-pkgs
+- name: Install fonts-khmeros-core package
+  apt: name={{ item }} state=latest
+  with_items:
+    - fonts-khmeros-core
```

Comment:
We actually don't need `fonts-khmeros-core`. I think it snuck into the list because when you `grep ros`, it ends up making it in. Sorry about that -- do you mind removing this bit?

Reply:
removed with commit 5bb24a2